docroot/themes/humsci/su_humsci_theme/scss/base/_global.scss

```diff
@@ -0,0 +1,24 @@
+@charset "UTF-8";
+
+//
+// Global
+//
+
+// Decanter
+@import 'decanter-no-markup';
+
+// SU HumSci
+@import
+  '../utilities/mixins/index';
+
+.horizontal-card {
```

Comment:
Why is this & `.vertical-card` getting added to global styles? these styles should be in the pattern.

Reply:
If I can't think of a logical defense, then it should be changed. :)

Reply:
Updated.